src/services/validator.js

```diff
@@ -1,10 +1,35 @@
 import Promise from 'bluebird';
 import yamlLint from 'yaml-lint';
-import { PensieveLinterError } from './errors';
+import jsyaml from 'js-yaml';
+import _ from 'lodash';
+import moment from 'moment';
```

Comment:
At this point I'm happy to bloat the repo, I'll create an issue to tackle moment size and come back to it later.